app/components/MainForm.js

```diff
@@ -108,304 +119,585 @@ const MainForm = (props) => {
 
-      {formSent ? <SuccessRegistration /> : (<>
-        <Sidebar id="sidebar" title="WELCOME TO CMYK" />
-        <div className="sm:w-2/3 xl:w-4/5 sm:min-h-screen">
-          <Navbar className="className" />
-          <div className="flex flex-col items-center">
-            <img src="./assets/img/CMYK.png" alt="cmyk" className="cmyk" />
-            <form
-              onSubmit={formik.handleSubmit}
-              className="grid sm:grid-cols-1 sm:gap-y-5  lg:grid-cols-2 lg:gap-x-1 lg:gap-y-3 xl:gap-x-2 xl:gap-y-4 2xl:gap-x-3 2xl:gap-y-6"
-            >
-              {formik.touched.full_name && formik.errors.full_name ? (
-                <input
-                  className="bg-red-100 border-l-2 border-red-700 placeholder-black"
-                  placeholder={formik.errors.full_name}
-                  value={formik.values.full_name}
-                  onChange={formik.handleChange}
-                  onBlur={formik.handleBlur}
-                  type="text"
-                  id="full_name"
-                />
-              ) : (
-                  <input
-                    type="text"
-                    id="full_name"
-                    placeholder="Full Name"
-                    value={formik.values.full_name}
-                    onChange={formik.handleChange}
-                    onBlur={formik.handleBlur}
-                  />
-                )}
-
-              {formik.touched.email && formik.errors.email ? (
-                <input
-                  className="bg-red-100 border-l-2 border-red-700 placeholder-black"
-                  placeholder={formik.errors.email}
-                  value={formik.values.email}
-                  onChange={formik.handleChange}
-                  onBlur={formik.handleBlur}
-                  type="email"
-                  id="email"
-                />
-              ) : (
-                  <input
-                    type="email"
-                    id="email"
-                    placeholder="E-mail adress"
-                    value={formik.values.email}
-                    onChange={formik.handleChange}
-                    onBlur={formik.handleBlur}
-                  />
-                )}
+            {data &&
+            data.allUsers.filter((role) => role.role === "Participant").length >
+              20 ? (
+              <div className="flex flex-col items-center">
+                <h1 className="text-gray-800 sm:text-lg md:text-xl xl:text-2xl mt-12 pt-2  mb-4">
+                  Sorry 😕
+                </h1>
+                <h1 className="text-gray-800  text-base xl:text-lg  ">
+                  We regret to inform you that there are no places available,
+                  but...🤔
+                </h1>
+                <p className="text-gray-500 font-bold text-base mr-20 ml-20 pr-22 pl-22 mb-10 ">
+                  You can still sign up as a substitute and in case there is a
+                  space available, we will notify you! 😊
```

Comment:
Maybe we can say here that, in case they don't know how substitutes role works, they can check the FAQ page